### PR TITLE
pagination.html to show all the page numbers instead of current/total

### PR DESCRIPTION
**Pull request by Anivarth**
Changes made to the simple template/pagination.html so that it will show all the page numbers instead of the current page and total number of pages Ex: <-1 2 3 4 5 6 -> instead of <-2/5 ->

This code supports upto 99999 pages. I am new to pelican, any suggestions are welcome. Thank you.

### CODE REVIEW

**Comment by justinmayer** [comment]
Thank you for your contribution, @Anivarth.

Can I get some input from @getpelican/reviewers regarding the perceived value this pull request would provide to the project?

File: pelican/themes/simple/templates/pagination.html [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
+			{% if page+1 == articles_page.number %}
+			   	<a href="#!">{{ page+1 }}</a>
+			{% else %}
+			    {% if page+1 == 1 %}
+			    	<a href="{{ SITEURL }}/{{ url_for_this_page }}.html">{{ page+1 }}</a>
+			    {% else %}
+			    	<a href="{{ SITEURL }}/{{ url_for_this_page }}{{ page+1 }}.html">{{ page+1 }}</a>
+			    {% endif %}
+		    {% endif %}
+	    {%- endfor %}
```
Comment (iKevinY) [inline pelican/themes/simple/templates/pagination.html:29]: Regardless of whether we think the level of granularity of pagination should be increased or not, I'm a bit hesitant on this method of constructing the other page URLs. From my initial reading of it, it's trying to reconstruct the URLs of other pages based on the current page (omitting the current page number)? The hard-coded `.html` in the final URLs makes me believe that change assumes some structure to the site that may or may not actually be true.

Comment (avaris) [reply pelican/themes/simple/templates/pagination.html:29]: I agree. Getting a page (and its `url`) is pretty straightforward:
```jinja2
articles_paginator.page(page_no).url
```

**Comment by ingwinlu** [comment]
Can we close this as there is apparently no interest in following it up?

**Comment by justinmayer** [comment]
@Anivarth: If you would like to re-compose this endeavor as a plugin, I think that would probably be the best approach.

Thanks to @iKevinY, @avaris, and @ingwinlu for reviewing.

**Pull request closed without merging**